app/scripts/views/settings/account_recovery/confirm_password.js

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import AuthErrors from '../../../lib/auth-errors';
+import Cocktail from 'cocktail';
+import FormView from '../../form';
+import PasswordMixin from '../../mixins/password-mixin';
+import ModalSettingsPanelMixin from '../../mixins/modal-settings-panel-mixin';
+import Template from 'templates/settings/account_recovery/confirm_password.mustache';
+
+const PASSWORD_SELECTOR = '#password';
+
+const View = FormView.extend({
+  template: Template,
+  className: 'account-recovery-confirm-password',
+  viewName: 'settings.account-recovery.confirm-password',
+
+  events: {
```

Comment:
The comment about [extending FormView's events](https://github.com/mozilla/fxa-content-server/pull/6392/files#r206087360) applies here too.

Reply:
@shane-tomlinson Not sure I follow what you mean, should it be extended like [this](https://github.com/mozilla/fxa-content-server/blob/3573ecf044f1774e4bdfae736f126f1ecad1a8f8/app/scripts/views/force_auth.js#L157)?